### PR TITLE
Update Maya token list

### PR DESCRIPTION
**Pull request by GiMa-Maya**

### CODE REVIEW

File: packages/toolboxes/cosmos/src/toolbox/kujira.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const KujiraToolbox = ({ server }: ToolboxParams = {}): KujiraToolboxType
           .filter(({ denom }) => {
             if (!denom || denom.includes("IBC/")) return false;
 
-            return denom === USK_KUJIRA_FACTORY_DENOM || !denom.startsWith("FACTORY");
+            return denom === USK_KUJIRA_FACTORY_DENOM || YUM_KUJIRA_FACTORY_DENOM || !denom.startsWith("FACTORY");
```
Comment (chillios-dev) [inline packages/toolboxes/cosmos/src/toolbox/kujira.ts:62]: Code doesn't work that way - it will return true always because of that

**Review by chillios-dev — CHANGES_REQUESTED** [review]

File: packages/toolboxes/cosmos/src/toolbox/kujira.ts (hunk 1 above)
Comment (GiMa-Maya) [reply packages/toolboxes/cosmos/src/toolbox/kujira.ts:62]: How would you do it? This part was quite vague to me

Comment (chillios-dev) [reply packages/toolboxes/cosmos/src/toolbox/kujira.ts:62]: @GiMa-Maya that needs to be coded as previous comparison `denom ===` 😅 that's how JS/TS works

**Comment by chillios-dev** [comment]
@GiMa-Maya -> I've updated and cherrypicked your commits to branch #729

**Pull request closed without merging**